src/test/config.ts

```diff
@@ -0,0 +1,31 @@
+// better handling of unhandled exceptions
+process.on('unhandledRejection', reason => {
+    throw reason;
+  });
+
+  export const config = {
+    TEST_URL: process.env.TEST_URL || 'http://localhost:${{ values.http_port }}',
```

Comment:
this repo has a simpler templating method, https://github.com/hmcts/expressjs-template/blob/master/bin/init.sh#L22
```suggestion
    TEST_URL: process.env.TEST_URL || 'http://localhost:3100',
```